internal/rbd/rbd_journal.go

```diff
@@ -216,7 +217,8 @@ func (rv *rbdVolume) Exists(ctx context.Context) (bool, error) {
 	// Fetch on-disk image attributes and compare against request
 	err = rv.getImageInfo()
 	if err != nil {
-		if _, ok := err.(ErrImageNotFound); ok {
+		var einf *ErrImageNotFound
```

Comment:
ErrImageNotFound is currently a type, do you suggest to make it a variable instead?

Reply:
is it possible to have a new function returns this struct pointer?

Reply:
That is what `errors.As()` does? Because errors can have several wrapped errors, just type-casting to the 'upper' error may not be sufficient. Internally `errors.As()` uses the `(err *error) Unwrap() error` method to go  through the list of wrapped errors.